standard/league_icon.lua

```diff
@@ -61,7 +63,7 @@ function LeagueIcon._make(icon, iconDark, link, name, size)
 		:wikitext('[[File:' .. icon .. imageOptions)
 	local darkSpan = mw.html.create('span')
 		:addClass('league-icon-small-image darkmode')
-		:wikitext('[[File:' .. icon .. imageOptions)
+		:wikitext('[[File:' .. iconDark .. imageOptions)
```

Comment:
This change seems unrelated to this PR. is it correct?

Reply:
yeah it is correct, it is a typo in the live version
(atm it is basically irrelevant as we do not yet have icondark added in main LeagueIconSmall templates and we do can not yet store/query it from lpdb)

Reply:
Please add to description

Reply:
ah wait the typo fix is already live, its just rebase, my bad